chainregistry.go

```diff
@@ -360,15 +364,19 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 				return nil, err
 			}
 		} else if cfg.Litecoin.Active && !cfg.Litecoin.RegTest {
-			ltndLog.Infof("Initializing litecoind backed fee estimator")
+			ltndLog.Infof(
```

Comment:
Nit: logs have a different wrapping so it should look like so (if the line exceeds 80 chars, split up the string):

```
ltndLog.Infof("Initializing litecoind backed fee estimator in %s mode",
        bitcoindMode.EstimateMode)
```